.gitignore

```diff
@@ -100,3 +100,4 @@ tags
 .idea/
 # test errors
 errors
+go.work*
```

Comment:
can we make the  golang workspace (`go.work`) permanent?

Reply:
I tried committing `go.work`, but it causes `make build` to fail. I think we'll need to investigate it separately and might require updating dependencies.